lib/applications/routes.js

```diff
@@ -4,53 +4,322 @@
  * @fileoverview Applications routing functions
  *
  */
+const config = require('config');
+const _ = require('lodash');
+const types = require('punchcard-content-types');
+const uuid = require('uuid');
+
+const database = require('../database');
+const model = require('./model');
+const utils = require('../utils');
 
 /**
  * All Webhooks Landing page
  * @param {object} req - HTTP Request
  * @param {object} res - HTTP Response
  * @param {object} next - Express callback
  *
+
+ * @returns {promise} promise wrapping database promise
  */
-const all = () => {};
+const all = (req, res, next) => {
+  return new Promise((resolve, reject) => {
+    return database
+      .select('*')
+      .from(`${config.applications.base}`)
+      .orderBy('name', 'DESC').then(rws => {
+        // add itentifier to each row
+        const rows = utils.routes.identifier(rws, req.applications.merged);
```

Comment:
This is _not_ our content. Why are you running this through the route identifier instead of just using the `name` column readily available in the rows you're going to be retrieving?